src/components/shared/ParamField.vue

```diff
@@ -36,11 +45,12 @@ const isArray = computed<boolean>(
   () => methodInfo.value.type?.match(/\[\]$/) != null
 )
 
-function makeWei(value: string, hex = false) {
+function makeWei(_value: string, hex = false) {
   try {
-    if (typeof value !== 'string') {
-      value = numberToHex(value)
-    }
+    const value =
+      typeof _value === 'string' && /^0x/.test(_value)
```

Comment:
is it to test if it starts with?
Maybe it can be re-written like this?

```suggestion
      typeof _value === 'string' && _value.startsWith("0x")
```

Reply:
Nice, yes.